UPGRADE-4.2.md

```diff
@@ -285,6 +285,23 @@ Messenger
    ```
  * The `EncoderInterface` and `DecoderInterface` interfaces have been replaced by a unified `Symfony\Component\Messenger\Transport\Serialization\SerializerInterface`.
    Each interface method have been merged untouched into the `Serializer` interface, so you can simply merge your two implementations together and implement the new interface.
+ * The `HandlerLocator` class was replaced with `Symfony\Component\Messenger\Handler\HandlersLocator`:
```

Comment:
should end with a dot (`.`) instead of `:` and end with a newline.

Reply:
done